terraform/modules/admin-lb/main.tf

```diff
@@ -30,13 +30,37 @@ resource "kubernetes_manifest" "admin-server-cert" {
   }
 }
 
+resource "null_resource" "wait_for_server_cert_creation" {
+  provisioner "local-exec" {
+    command = <<-EOT
+    COUNTER=0
+    MAX_TRIES=50
```

Comment:
Make this (and the sleep time, see my other comment) dynamic to allow the user to set customized values.

Example: `MAX_TRIES=${MAX_TRIES:-0}` if you want to use environment variables or `MAX_TRIES=${1:-0}` if you want to use command arguments.